src/utilities/dashboardEvents.js

```diff
@@ -159,6 +148,13 @@ export const checkIfDatesFallOnWeekend = (start, end) => {
   }
 };
 
+export const checkSameMonth = (dateOne, dateTwo) => {
```

Comment:
This is more like a date extension than a dashboard event.

Reply:
Moved to `dates.js`